src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,12 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService userService = new UserService();
```

Comment:
creation of `userService` should be on method level? We don't need to create on every call of `login` a `new UserService();`. Think about it please!

Reply:
Can I make the method from non-static to static to call it from the class?